### ISSUE TITLE
undefined references while building with mingw32 on linux: _imp__ohmd_list_open_device_s and  _imp__ohmd_sleep

### ISSUE THREAD

**Issue by HickDead** [OpenHMD/OpenHMD#245]
While trying to build for windows on Debian with mingw32, I'm getting the following error: 
```  CCLD     unittests.exe
/usr/bin/i686-w64-mingw32-ld: ../../src/.libs/libopenhmd.a(libopenhmd_la-openhmd.o): in function `ohmd_list_open_device':
/tmp/OpenHMD/src/openhmd.c:257: undefined reference to `_imp__ohmd_list_open_device_s'
/usr/bin/i686-w64-mingw32-ld: ../../src/.libs/libopenhmd.a(libopenhmd_la-platform-win32.o): in function `ohmd_destroy_thread':
/tmp/OpenHMD/src/platform-win32.c:89: undefined reference to `_imp__ohmd_sleep'
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:400: unittests.exe] Error 1
make[3]: Leaving directory '/tmp/OpenHMD/tests/unittests'
make[2]: *** [Makefile:361: all-recursive] Error 1
make[2]: Leaving directory '/tmp/OpenHMD/tests'
make[1]: *** [Makefile:463: all-recursive] Error 1
make[1]: Leaving directory '/tmp/OpenHMD'
make: *** [Makefile:372: all] Error 2
```
Hidapi was built with mingw32 and installed in the $PREFIX dir beforehand.  And I'm using the exact commands from the README.md (after running autogen.sh since I'm building from the repo).

Am I doing something wrong here?  Building natively for linux works just fine.

**Comment by HickDead** [comment]
Apparently this has something to do with being unaware of linking statically, as adding ``CFLAGS="-DOHMD_STATIC" `` in front of the ./configure line seems to help.  So much for not having to worry if you're unsure...

Meanwhile I have turned to building on windows with MSYS2/MinGW natively and using cmake, which works like a charm (except for libhidapi-0.dll still getting linked dynamically).  I have documented the process in the Relativty wiki: https://wiki.relativty.net/index.php/OpenHMD.
